pkg/wait/wait_for_ready.go

```diff
@@ -16,14 +16,15 @@ package wait
 
 import (
 	"fmt"
-	"github.com/knative/pkg/apis"
 	"io"
+	"time"
+
```

Comment:
Also not related to this PR, I see where this comes from. As this PR was rebased on #186 which introduces automatic import reformatting, you can't avoid this here.

This kind of bogus changes will/can happen on all existing PRs but should be gone afterwards.

I'm fine with that.